README.md

```diff
@@ -0,0 +1,225 @@
+
+A good example of ConfigMaps that can be used can be found in our [dev-tools](https://github.com/openstack-k8s-operators/osp-director-dev-tools) GitHub project. Direct links for each the sample files can be found below:
+
+-[Net-Config files](https://github.com/openstack-k8s-operators/osp-director-dev-tools/tree/master/ansible/files/osp/net_config)
+
+-[Tripleo Deploy custom files](https://github.com/openstack-k8s-operators/osp-director-dev-tools/tree/master/ansible/templates/osp/tripleo_deploy) (NOTE: these are Ansible templates and need to have variables replaced to be used directly!)
+
+Once you customize the above template/examples for your environment you can create configmaps for both the 'tripleo-deploy-config-custom' and 'tripleo-net-config' ConfigMaps by using these example commands on the files containing each respective configmap type (one directory for each type of configmap):
+
+```bash
+# create the configmap for tripleo-deploy-config-custom
+oc create configmap -n openstack tripleo-deploy-config-custom --from-file=tripleo-deploy-config-custom/ --dry-run -o yaml | oc apply -f -
+
+# create the configmap for netconfig
+oc create configmap -n openstack tripleo-net-config --from-file=net-config/ --dry-run -o yaml | oc apply -f -
+```
+
+3) Create a [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) for your OpenStackControlPlane. This secret will provide the default password for your OSP Controller VMs.
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: userpassword
+  namespace: openstack
+data:
+  # 12345678
+  NodeRootPassword: MTIzNDU2Nzg=
+```
+
+If you write the above YAML into a file called ctlplane-secret.yaml you can create the Secret via this command:
+
+```bash
+oc create -f ctlplane-secret.yaml
+```
+
+3) Define your OpenStackControlPlane custom resource. The OpenStackControlPlane custom resource provides a cental place to create and scale VMs used for the OSP Controllers along with any additional vmsets for your deployment. At least 1 Controller VM is required for a basic demo installation and per OSP High Availability guidelines 3 Controller VMs are recommended.
+
+```yaml
+apiVersion: osp-director.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: overcloud
+  namespace: openstack
+spec:
+  openStackClientImageURL: quay.io/openstack-k8s-operators/tripleo-deploy:latest
+  passwordSecret: userpassword
+  virtualMachineRoles:
+    controller:
+      roleName: Controller
+      roleCount: 1
+      networks:
+        - ctlplane
+      cores: 6
+      memory: 12
+      diskSize: 40
+      baseImageURL: http://download.devel.redhat.com/brewroot/packages/rhel-guest-image/8.3/417/images/rhel-guest-image-8.3-417.x86_64.qcow2
+      baseImageVolumeName: openstack-base-img
+      storageClass: host-nfs-storageclass
```

Comment:
since we now merged isolated networks, this needs to be updated and maybe we want to use isolated networks per default?

~~~
apiVersion: osp-director.openstack.org/v1beta1
kind: OpenStackControlPlane
metadata:
  name: overcloud
  namespace: openstack
spec:
  openStackClientImageURL: quay.io/openstack-k8s-operators/tripleo-deploy:16.2_20210309.1
  openStackClientNetworks:
        - ctlplane
        - external
        - internalapi
  passwordSecret: userpassword
  virtualMachineRoles:
    controller:
      roleName: Controller
      roleCount: 3
      networks:
        - ctlplane
        - internalapi
        - external
        - tenant
        - storage
        - storagemgmt
      cores: 6
      memory: 12
      diskSize: 40
      # FIXME: remove this once we make baseImageURL optional
      baseImageURL: http://download.devel.redhat.com/brewroot/packages/rhel-guest-image/8.3/417/images/rhel-guest-image-8.3-417.x86_64.qcow2
      baseImageVolumeName: openstack-base-img
      storageClass: host-nfs-storageclass
~~~